app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -46,12 +49,27 @@ protected void onCreate(Bundle savedInstanceState) {
         okButton = (Button) findViewById(R.id.okButton);
 
         sharedpreferences = getSharedPreferences(MyPREFERENCES, Context.MODE_PRIVATE);
+        editor = sharedpreferences.edit();
+
+        comrade1editText.setText(sharedpreferences.getString(comrade1, ""));
+        comrade2editText.setText(sharedpreferences.getString(comrade2, ""));
+        comrade3editText.setText(sharedpreferences.getString(comrade3, ""));
+        comrade4editText.setText(sharedpreferences.getString(comrade4, ""));
+        comrade5editText.setText(sharedpreferences.getString(comrade5, ""));
+
         okButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Toast.makeText(getApplicationContext(),"Comrades phone numbers has been updated", Toast.LENGTH_LONG).show();
+                editor.putString(comrade1, comrade1editText.getText().toString());
+                editor.putString(comrade2, comrade2editText.getText().toString());
+                editor.putString(comrade3, comrade3editText.getText().toString());
+                editor.putString(comrade4, comrade4editText.getText().toString());
+                editor.putString(comrade5, comrade5editText.getText().toString());
+                editor.putString(comrade6, comrade6editText.getText().toString());
+
+                editor.commit();
+                Toast.makeText(getApplicationContext(), "Comrades phone numbers has been updated", Toast.LENGTH_LONG).show();
```

Comment:
Please add this string to the string.xml as well. The standard is you don't use any hard coded strings in the code itself. You add all the strings to string.xml and refer from there.